### PR TITLE
Support PHP 8.0

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Runs HHVM and PHP 5 tests on travis. Runs PHP 7 and 8 on GitHub Actions.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Hmm, latest PHP 7.3 seems to have a bug causing infinite recursion...

**Comment by GrahamCampbell** [comment]
Tests passing now. Travis run: . Actions run: https://github.com/GrahamCampbell/whoops/actions/runs/298433086.

NB @filp Travis is not showing up in the status because you need to switch to travis-ci.com from .org. Github actions will only start to show up after this PR is merged.

File: tests/Whoops/RunTest.php [hunk 1]
```diff
@@ -402,6 +406,7 @@ public function testSilenceErrorsInPaths()
 
     /**
      * @covers Whoops\Run::handleError
+     * @requires PHP < 8
```
Comment (staabm) [inline tests/Whoops/RunTest.php:409]: Why does this not work on php8

Comment (GrahamCampbell) [reply tests/Whoops/RunTest.php:409]: Because this is a different kind of error now.

Comment (GrahamCampbell) [reply tests/Whoops/RunTest.php:409]: PHP does not permit silencing it.

**Review by denis-sokolov — APPROVED** [review]
Thanks so much, Graham! Good effort has been put in here.

**Comment by GrahamCampbell** [comment]
🎉

**Pull request merged**